docs/Antora/modules/howto_guides/pages/howto_configure_streaming.adoc

```diff
@@ -0,0 +1,298 @@
+the amount of hops between Devices the data must make.
+* "Fallback" mode (ID 2) - is used to establish all possible Streaming connections.
+* "Not connected" mode (ID 3) - with this set the information about supported streaming protocols published
+by the Device is not used to automatically establish Streaming connections.
+
+[tabs]
+====
+Cpp::
++
+[source,cpp]
+----
+#include <opendaq/opendaq.h>
+
+using namespace daq;
+
+int main()
+{
+    // Create a new Instance that we will use for all the interactions with the SDK
+    daq::InstancePtr instance = daq::Instance(MODULE_PATH);
+
+    // Get the default configuration Property object for OPC UA enabled device type
+    daq::PropertyObjectPtr deviceConfig = instance.getAvailableDeviceTypes().get("daq.opcua").createDefaultConfig();
+
+    // Allow multiple streaming protocols by device configuration
+    deviceConfig.setPropertyValue("AllowedStreamingProtocols", daq::List<daq::IString>("daq.ns", "daq.wss"));
+
+    // Set websocket streaming protocol as primary by device configuration
+    deviceConfig.setPropertyValue("PrimaryStreamingProtocol", "daq.wss");
+
+    // Disregard direct streaming connections for nested devices,
+    // establish the minimum number of streaming connections possible.
+    deviceConfig.setPropertyValue("StreamingConnectionHeuristic", 0);  
+
+    // Find and connect to a device hosting an OPC UA TMS server
+    const auto availableDevices = instance.getAvailableDevices();
+    daq::DevicePtr device;
+    for (const auto& deviceInfo : availableDevices)
+    {
+        if (deviceInfo.getConnectionString().toView().find("daq.opcua://") != std::string::npos)
+        {
+            // Add device using modified configuration Property object
+            device = instance.addDevice(deviceInfo.getConnectionString(), deviceConfig);
+            break;
+        }
+    }
+
+    if (!device.assigned())
+        std::cerr << "No relevant device found!" << std::endl;
+    else
+        // Output the name of the added device
+        std::cout << device.getInfo().getName() << std::endl;
+    
+    return 0;
+}
+----
+====
+
+[#streaming_config_per_signal]
+== Configure Streaming per Signal
+
+Once the xref:glossary:glossary.adoc#device[Device] is connected, the Streaming sources of its
+xref:glossary:glossary.adoc#signal[Signal]s can be examined and modified for each signal individually
+at any given time.
+
+The Streaming sources are identified by a connection string that includes the protocol prefix, indicating
+the protocol type ID, and parameters based on the protocol type (IP address, port number etc.).
+To manipulate the Streaming sources of particular Signal the `MirroredSignalConfig` object is used,
+it provides ability to:
+
+* retrieve a list of streaming sources available for signal,
+* get the currently active streaming source,
+* modify the active source or unset it to disable data streaming for a signal.
```

Comment:
I'd mention the `ISignal::setStreamed` function here.

Reply:
setStreamed / getStreamed functions mention added